src/Misc/layoutroot/run.sh

```diff
@@ -1,15 +1,27 @@
 #!/bin/bash
 
+# Validate not sudo
 user_id=`id -u`
-
 if [ $user_id -eq 0 ]; then
     echo "Must not run interactively with sudo"
     exit 1
 fi
 
-if [ ! -f .agent ] && [[ "$1" != "--yaml" ]] ; then
+# Get the agent root directory - https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within
```

Comment:
The script previously assumed your working directory was the agent root dir. With local run the working directory must not be the agent root dir.